jose-jwt/jose-jwt.csproj

```diff
@@ -93,7 +93,7 @@ RFC 7797:
     <NeutralLanguage>en-US</NeutralLanguage>
     <VersionPrefix>3.0.0</VersionPrefix>
     <Authors>Dmitry Vsekhvalnov</Authors>
-    <TargetFrameworks>net40;net461;netstandard1.4;netstandard2.1</TargetFrameworks>
+    <TargetFrameworks>netstandard1.4;netstandard2.1</TargetFrameworks>
```

Comment:
let's keep `net40`, don't believe it is part of any .netstandard.

Was it causing any build issues with new changes?

Reply:
- I don't have any environment to test net40. I believe it went out of support by MS in 2016... is it really required to maintain support for this on the master branch / new releases? (Could you security patch fix a net40 branch if requried?).
- Note I've done more work testing net461 and have today made further commit to get the net461 unit test project clean. - Also I added 461 as a target of the 'main' unit test project and fixed some existing breakages to see how well the main project runs against 461.
- I think for net40 though I wouldn't get any time allowed to validate.

Reply:
i can test .net40, no problem.

Reply:
@gjermystreeva, looks like final one and we can merge it.

Reply:
.net40 added back now